<?xml version="1.0"?>
<launch>

    <node pkg="exercise6" type="map_goalz" name="map_goals" output="screen">
    <remap from="/map_goals" to="/move_base_simple/goal"/>
    
    </node>

</launch>
